packages/fx-core/src/core/index.ts

```diff
@@ -588,6 +588,22 @@ class CoreImpl implements Core {
               icon: "M365",
             },
           ]);
+
+          this.ctx?.telemetryReporter?.sendTelemetryEvent(TelemetryEvent.Login, {
```

Comment:
Why are you sending the telemetry in commonlib for cli but in core for extension?

Reply:
Also, the callback here will only be triggered when user click sign in from sidebar, how about other entries(webview, command palette and lazy sign in)?

Reply:
OK, change to another place.